docs/core/integration-quality-scale/rules/iqs016.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "IQS016 - Entities are assigned an appropriate EntityCategory"
+---
+
+IQS016 is as follows:
+
+Entities are assigned an appropriate EntityCategory
+
+## Reasoning
+
+Entities should be assigned an appropriate EntityCategory to ensure that they are correctly classified and can be easily identified.
```

Comment:
Maybe clarify that the default entity category `None` is also sometimes the appropriate answer.